generator/src/cmd/generate.go

```diff
@@ -18,20 +18,116 @@ package cmd
 import (
 	"application-generator/src/pkg/generate"
 	"github.com/spf13/cobra"
+	"application-generator/src/pkg/model"
```

Comment:
As we talked, we need a controller(getter method) instead of importing the models directly.